pegasus/sites.v3/br.code.org/views/header.haml

```diff
@@ -13,8 +13,10 @@
         %img{:src=>"/images/twitter_purple.png"}
       %a{:href=>"http://codeorg.tumblr.com/", :title=>'Tumblr', :target=>"_blank", :style=>"text-decoration:none"}
         %img{:src=>"/images/tumblr_purple.png"}
-      %a{:href=>"http://zazzle.com/codeorg", :title=>'Zazzle', :target=>"_blank", :style=>"text-decoration:none"}
-        %img{:src=>"/images/zazzle_purple.png"}
+      %a{:href=>"https://www.instagram.com/codeorg/", :title=>'Instagram', :target=>"_blank", :style=>"text-decoration:none"}
```

Comment:
nit: Any reason not to remove the trailing slash in the `href` be removed? Save a byte. :) Also below (for the store) and with other languages.